_posts/2019-12-19-sw-ix-review.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: Star Wars Episode IX The Rise of Skywalker Review [SPOILERS]
+date: 2019-12-19 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[Obviously heavy spoilers for the movie and SW 1 to 9]
+
+Coming in I had really love expectations, unfortunately that was not enough.
+
+The movie at the same level of shittiness as [The Force Awakens][1] and way worse than The Last Jedi. It seems like Disney doesn't care at all about existing lore.
+
+They made a plot and rammed it through not caring about existing characters arcs or existing lore. I see a lot of similarities to Game Of Thrones Season 8 in there, but it is not as bad as GoT in my opinion.
+
+Where to start... so many things wrong with this movie.
+
+## The whole SW story arc is nullified
+
+George Lucas has said repeatedly that the whole arc of SW 1 to 6 was the fall to Anakyn to the dark side and its redemption and even if you don't like the prequels (I love them) you can see how they are consistent with the overarching story and rules and how they deliver on that vision. The whole reason for his fall was trying to save Padme. The new trilogy now make it seem that bringing someone back from death is a joke.
+
+The whole idea of the chosen one that would bring balance to the force that went vindicated in episode 6 is now in shambles. Anakyn in the end did not bring balance. He failed to kill the emperor. What a joke.
+
+## No bigger plot
+
+SW 7 to 9 don't have an overline structure. They just decided 3 plots for the 3 movies and that's it. They have characters in common, are set in the same universe, and they happen chronologically but apart from that there is not much else going on.
+
+Many story arcs started in previous movies were never finished and were just left abandoned (Fin story arc for example) and you can't see an overall story progression from 7 to 9 as you do in 1 to 3 and 4 to 6.
+
+In the end we are at the end of 9 in the same place as we were in 6, the story didn't not progress at all. Apart from the fact that most people we cared about are now dead. (RIP Carrie Fisher)
+
+## Force as a superpower
+
+The force has always been very strong in what it could do, but it had its limitations. We always knew what it could do (pull, push, lightnings, ..) and what it could not (teleport). Already in The Last Jedi we've seen the force gaining the ability of making someone appearing on another planet and interact with people and things there. We've seen force ghosts before so this could be excused and I didn't think too much ill of it. In this movie those new abilities just keep piling up and up in an unbeliavable fashion. The authors seem to excuse themselves behind the curtain of "it's because of the bond between Kylo and Rei", but even with the bond this is totally crazy.
```

Comment:
appearing - > appear

Reply:
Rei? R E Y

Reply:
These kinds of bonds usually were with master/padawan... something that took time to develop... not just two random people -. -